mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -48,6 +48,18 @@ def __init__(self, view: 'SpectrumViewerWindowView', main_window: 'MainWindowVie
         self.main_window = main_window
         self.model = SpectrumViewerWindowModel(self)
         self.export_mode = ExportMode.ROI_MODE
+        self.main_window.stack_changed.connect(self.handle_stack_changed)
+
+    def handle_stack_changed(self):
```

Comment:
Can this have a ` -> None` type annotation.
Looks likes adding it will force an extra None check. (or adding an assert if you don't think it can ever actually be None here).